src/main/java/redis/clients/jedis/commands/MultiKeyCommandsPipeline.java

```diff
@@ -32,15 +32,24 @@
 
   Response<String> lmove(String srcKey, String dstKey, ListDirection from, ListDirection to);
 
-  Response<String> blmove(String srcKey, String dstKey, ListDirection from, ListDirection to, int timeout);
+  Response<String> blmove(String srcKey, String dstKey, ListDirection from, ListDirection to,
+      double timeout);
 
   Response<List<String>> blpop(String... args);
 
+  Response<List<String>> blpop(int timeout, String... args);
+
+  Response<KeyedListElement> blpop(double timeout, String... args);
+
   Response<List<String>> brpop(String... args);
 
-  Response<KeyedTuple> bzpopmax(int timeout, String... keys);
+  Response<List<String>> brpop(int timeout, String... args);
```

Comment:
@sazzad16 I think you should deprecate the interface methods and not only the implementation methods

Reply:
@gkorland The deprecated methods were not part of any interface.